src/app/zap-templates/zcl/data-model/chip/general-diagnostics-cluster.xml

```diff
@@ -117,7 +117,7 @@ limitations under the License.
     </event>
     <event side="server" code="0x03" name="BootReason" priority="critical" optional="false">
       <description>Indicate the reason that caused the device to start-up.</description>
-      <field id="0" name="BootReason" type="BootReasonType"/>
+      <field id="0" name="BootReason" type="BootReasonEnum"/>
```

Comment:
This says enum suffix but we have RadioFault and other Fault without Enum. I guess that's the spec but this lack of symmetry is also unfortunate. I wonder if it's worth fixing in spec

Reply:
I would be OK trying to fix this in spec, if we think that change would be taken....